ansible/roles/fluentd-node-collector/templates/fluentd-daemonset.yaml

```diff
@@ -123,17 +119,10 @@ items:
     name: MASTER_URL
     value: "https://kubernetes.default.svc.cluster.local"
   -
-    description: "Hostname (or IP) for reaching fluentd normalizer to write logs"
-    name: FLUENTD_NORMALIZER
-    value: "logging-es"
-  -
-    description: "Port number for reaching fluentd normalizer to write cluster logs"
-    name: FLUENTD_NORMALIZER_PORT
-    value: "9200"
-  -
     description: "The prefix of the image to use."
     name: IMAGE_PREFIX
-    value: "atomic-registry.usersys.redhat.com:5000/t0ffel/"
+    value: "docker.io/t0ffel/"
```

Comment:
You might want to consider putting these choices in the README for folks to track down.